Source/Core/Core/BootManager.cpp

```diff
@@ -217,6 +218,25 @@ static GPUDeterminismMode ParseGPUDeterminismMode(const std::string& mode)
   return GPU_DETERMINISM_AUTO;
 }
 
+static constexpr u16 BT_INFO_SECTION_LENGTH = 0x460;
```

Comment:
Note that this constant is already defined in `WII_IPC_HLE_Device_usb_bt_base.h`. These are also both not really related to actually booting the system itself. You can just implement `RestoreBTInfoSection` in `WII_IPC_HLE_Device_usb_bt_base.h` alongside the `BackupBTInfoSection` to keep both functions together, and then just expose that function in the header.